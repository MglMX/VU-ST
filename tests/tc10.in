1
u
u
exit
