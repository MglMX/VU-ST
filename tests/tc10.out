Choose category to play or type exit to leave
(1) Cars
(2) Animals
Type category number:==============
Category: Cars
Letters: []
Lives left: 6
Word to guess: _ _ _ _ _ _
==============
Category: Cars
Letters: ['u']
Lives left: 6
Word to guess: _ u _ u _ _
You've already checked for this letter!
Thank you for playing!
